### PR TITLE
Added dynamic I2C mini motor address support, and update uwp library

### PR DESCRIPTION
**Pull request by awong1900**
@lanselambor @DexterInd Update the driver

### CODE REVIEW

File: Software/CSharp/GrovePi/DeviceFactory.cs [hunk 1]
```diff
@@ -47,11 +48,11 @@ internal class DeviceBuilder : IBuildGroveDevices
     {
         private const string I2CName = "I2C1"; /* For Raspberry Pi 2, use I2C1 */
         private const byte GrovePiAddress = 0x04;
-        private const byte DisplayRgbI2CAddress = 0x62;
-        private const byte DisplayTextI2CAddress = 0x3e;
+        private const byte DisplayRgbI2CAddress = 0xC4;
```
Comment (DexterInd) [inline Software/CSharp/GrovePi/DeviceFactory.cs]: Why are we changing the addresses here?

File: Software/CSharp/GrovePi/DeviceFactory.cs [hunk 2]
```diff
@@ -47,11 +48,11 @@ internal class DeviceBuilder : IBuildGroveDevices
...
+        private const byte DisplayTextI2CAddress = 0x7C;
         private const byte SixAxisAccelerometerI2CAddress = 0x1e;
-        private const byte MiniMotorDriverI2cAddress1 = 0x62;  // 0xC4
-        private const byte MiniMotorDriverI2cAddress2 = 0x60;  // 0xC0
+        private const byte MiniMotorDriverCH1I2cAddress = 0xC4;
+        private const byte MiniMotorDriverCH2I2cAddress = 0xC0;
```
Comment (DexterInd) [inline Software/CSharp/GrovePi/DeviceFactory.cs]: Why are we renaming these variables?

File: Software/CSharp/GrovePi/DeviceFactory.cs (hunk 1 above)
Comment (awong1900) [inline Software/CSharp/GrovePi/DeviceFactory.cs]: Firstly, i found that mini motor's address is not same with [Wiki](http://wiki.seeed.cc/Grove-Mini_I2C_Motor_Driver_v1.0/#change-default-i2c-address). 
Then i found them have one address is same. So i think change it should be clear. 
refer to: https://github.com/Seeed-Studio/Grove_LCD_RGB_Backlight/blob/master/rgb_lcd.h  

BTW, seeed will be change default minimotr driver's i2c address later.

File: Software/CSharp/GrovePi/DeviceFactory.cs (hunk 2 above)
Comment (awong1900) [inline Software/CSharp/GrovePi/DeviceFactory.cs]: That will be clear to difference for channel 1 and channel 2.

**Pull request merged**